include/vcpkg/base/messages.h

```diff
@@ -1308,6 +1371,10 @@ namespace vcpkg
                     "'{value}' is either a tool name or a package name.",
                     "'{value}' failed while extracting {path}.");
     DECLARE_MESSAGE(PackageRootDir, (), "", "(Experimental) Specify the packages root directory.");
+    DECLARE_MESSAGE(PackagesNotInstalled,
+                    (msg::package_name, msg::triplet, msg::path),
+                    "",
+                    "Packages not installed {package_name}:{triplet} {path}");
```

Comment:
I think we should have a generic "your vcpkg "installed" tree is corrupted" message and this should use that.